### DISCUSSION TITLE
How to get the force RMSE for different committee sizes ?

### DISCUSSION THREAD

**Discussion by YuliangS** [MarsalekGroup/aml#32] [Q&A] [answered]
Hi 
Sorry for the trouble.
I read the JCP paper (doi: 10.1063/5.0016004) published by your team. It was an excellent work.
I want to reproduce the bottom panel of Fig.3, which shows the the force root mean square error (RMSE) of the C-NNP model for different committee sizes from one to eight members.  But I can only found RMSE of single model.
![image](https://github.com/MarsalekGroup/aml/assets/105383439/d080df5e-a09d-4373-98de-e3916f4d7285)

Could you tell me how to do it?

Thank you very much!!

**Comment by cschran** [comment, marked as answer]
Hi @YuliangS,

Thanks for your interest in our work.
At the moment, you will need to do this in post processing - Repeat `QbC` and set `remove_output = False`. This will keep all individual predictions of the committee members. You can then afterwards read those in for each generation and do the averaging for different committee sizes by hand.
Note that we report the average of all possible splits in the above Figure, but it should not change much if you consider only one possibility per committee size.

All the best,
Christoph

**Comment by YuliangS** [comment]
Thank you so much!
